packages/jsxstyle-utils/src/getStyleKeysForProps.ts

```diff
@@ -52,14 +64,7 @@ export default function getStyleKeysForProps(props: any, pretty = false) {
   let usesMediaQueries = false;
 
   // return value
-  const styleKeyObj: {
-    [key: string]: {
-      styles: string;
-      mediaQuery?: string;
-      pseudoclass?: string;
-      pseudoelement?: string;
-    };
-  } & { classNameKey: string } = <any>{};
+  const styleKeyObj: StyleKeyObj = <any>{};
```

Comment:
Can you cast this to `StyleKeyObj` instead of `any`?

(I still think you should be using `as` for casting, but that's a style thing.)

Reply:
Yep, that works. Thanks!

Reply:
I might even call it `StyleKeyDict` to be consistent with your naming.